server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -461,6 +461,16 @@ public TSStatus executeBatchStatement(TSExecuteBatchStatementReq req) {
 
     InsertRowsPlan insertRowsPlan = new InsertRowsPlan();
     int index = 0;
+
+    CreateMultiTimeSeriesPlan multiPlan = new CreateMultiTimeSeriesPlan();
```

Comment:
you cannot create so many Object instances before you confirm some statements can be wrapped by a CreateMultiTimeSeriesPlan.

Reply:
Changes completed